CMakeLists.txt

```diff
@@ -180,6 +180,10 @@ if(valijson_BUILD_TESTS)
         set_target_properties(test_suite PROPERTIES COMPILE_FLAGS " -pedantic -Werror -Wshadow -Wunused")
     endif()
 
+    if (MSVC)
+        target_compile_options(test_suite PRIVATE "/bigobj")
```

Comment:
What does this actually do?

Reply:
https://docs.microsoft.com/en-us/cpp/build/reference/bigobj-increase-number-of-sections-in-dot-obj-file << allows more sections in the application. Without it the test application started failing to build on VS 2022 for me.

Reply:
Oh that's good to know!